### PR TITLE
added dessert menu

### PR DESCRIPTION
**Pull request by JitenWhatever**
I have added dessert menu .

### CODE REVIEW

File: src/main/java/org/tektutor/menuwebservice/MenuController.java [hunk 1]
```diff
@@ -52,6 +52,29 @@ public String sayHello(@RequestParam String firstName, @RequestParam String last
...
+	public HashMap<Integer, String> getDesserts(){
+		Hashmap<Integer, String> dessertList = new HashMap<Integer, String>();
+		dessertList.put(1, "Buttery Raspberry Crumble Bars");
+		dessertList.put(2, "Mint Oreo Cake");
+		dessertList.put(3, "Ultimate Gooey Brownies");
+		dessertList.put(4, "Chocolate And Strawberry Cake");
+		dessertList.put(5, "Coconut Cream Crepe Cake");
+		dessertList.put(6, "Butterfinger Cookie Dough Cheesecake Bars");
+		dessertList.put(7, "Pomegranate Mousse Cake");
+		dessertList.put(8, "Chocolate Chip Cookies With Nutella, Brown Butter And Sea Salt");
```
Comment (yogeshvar) [inline src/main/java/org/tektutor/menuwebservice/MenuController.java]: Instead of having this big dessert name, we can have some like “Nutella choco”

Comment (JitenWhatever) [reply src/main/java/org/tektutor/menuwebservice/MenuController.java]: ok, thanks

Comment (JitenWhatever) [reply src/main/java/org/tektutor/menuwebservice/MenuController.java]: Updated dessert names in dessert menu.

File: src/main/java/org/tektutor/menuwebservice/MenuController.java [hunk 2]
```diff
@@ -52,6 +52,29 @@ public String sayHello(@RequestParam String firstName, @RequestParam String last
...
+		dessertList.put(9, "Pavlova With Blueberry Jam");
+		dessertList.put(10, "Salted Caramel Chocolate Tart");
+		dessertList.put(11, "Banana Pudding Parfaits");
+		dessertList.put(12, "Browned Butter Toffee Chocolate Chip Cookies");
+		dessertList.put(13, "Nutella Mascarpone Chocolate Tarts");
```
Comment (yogeshvar) [inline src/main/java/org/tektutor/menuwebservice/MenuController.java]: This sounds great and perfect 😎

**Pull request merged**